### PR TITLE
CES-10988 

### PR DESCRIPTION
**Pull request by gaelrehault**
adding ironic python dependencies to undercloud host
python3 edits

### CODE REVIEW

File: src/pilot/config_idrac.py [hunk 1]
```diff
@@ -657,7 +657,7 @@ def config_idrac(instack_lock,
                     new_drac_client.is_idrac_ready()
                     password_changed = True
                 except exceptions.WSManInvalidResponse as ex:
-                    if "unauthorized" in ex.message.lower():
+                    if "unauthorized" in str(ex).lower():
```
Comment (heydog15) [inline src/pilot/config_idrac.py]: Why the change here?  Is WSMan changing?

Comment (gaelrehault) [reply src/pilot/config_idrac.py]: Python3 handling or exceptions changed

**Comment by heydog15** [comment]
Ok, thanks.
